app/models/patient.ts

```diff
@@ -6,18 +6,18 @@ export async function loadPatientDataByID(
   connection: Connection
 ) {
   const personCCC = await fetchPersonCCCByID(personId, connection);
-  console.log("ccc", personCCC);
-  return await loadPatientData(personCCC, connection);
+  console.log("ccc");
+  return await loadPatientData(personCCC[0].patient_ccc_number, connection);
```

Comment:
What happens when `personCCC[0]` is empty

Reply:
I Will put checks in place to handle that. Thanks